src/lib/messaging/channels/googlechat/runtime/hermes-adapter.py

```
@@ -0,0 +1,316 @@
+`_load_sa_credentials` and `_new_authed_http` are Hermes internals only because
+`PlatformEntry` carries no credential or transport field;
+`image-build-probes.py googlechat-override-seams` pins them so drift fails the
+image build instead of silently restoring the stock gRPC adapter.
+"""
+
+import asyncio
+import dataclasses
+import importlib.util
+import logging
+
+_GC_REST_PLACEHOLDER_TOKEN = "openshell:resolve:env:GOOGLE_CHAT_ACCESS_TOKEN"
```

Ruff (0.16.1)

[error] Possible hardcoded password assigned to: "_GC_REST_PLACEHOLDER_TOKEN"

(S105)